qt/python/mantidqtinterfaces/test/Muon/corrections_tab_widget/background_corrections_view_test.py

```diff
@@ -23,8 +23,6 @@
                                                                                                    STATUS_COLUMN_INDEX,
                                                                                                    SHOW_MATRIX_COLUMN_INDEX)
 
-from qtpy.QtWidgets import QApplication
-
 
 @start_qapplication
 class BackgroundCorrectionsViewTest(unittest.TestCase, QtWidgetFinder):
```

Comment:
Can we drop `QtWidgetFinder` from the inherited classes whilst we're here? 
In unit tests we shouldn't require Qt Widgets

I'd like to get rid of `@start_qapplication` but I think that's beyond the scope of this PR